river/tree/extremely_fast_decision_tree.py

```diff
@@ -43,6 +43,32 @@ class ExtremelyFastDecisionTreeClassifier(HoeffdingTreeClassifier):
     nominal_attributes
         List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
         should be treated as continuous.
+    attribute_observer
```

Comment:
Same comment as before.

Reply:
Updated in commit [d247eb7](https://github.com/creme-ml/creme/pull/379/commits/d247eb7b873f17442e35e1b1df8054ad05611a4b).